modules/core/src/va_intel.cpp

```diff
@@ -137,7 +141,13 @@ Context& initializeContextFromVA(VADisplay display, bool tryInterop)
             contextInitialized = true;
 
             cl_platform_id platform = platforms[found];
-            std::string platformName = PlatformInfo(platform).name();
```

Comment:
Please replace `platform` => `&platform`.

With verbose replacements it is better to keep original code (to avoid replacement back):
```
#if 0  // code doesn't works due to ...
   original code
#else
  ... verbose code ...
#endif
```